src/box/sql/expr.c

```diff
@@ -113,7 +113,7 @@ sql_expr_type(struct Expr *pExpr)
 		assert(pExpr->pRight != NULL && pExpr->pLeft != NULL);
 		enum field_type lhs_type = sql_expr_type(pExpr->pLeft);
 		enum field_type rhs_type = sql_expr_type(pExpr->pRight);
-		return sql_type_result(rhs_type, lhs_type);
+		return sql_type_result(lhs_type, rhs_type);
```

Comment:
There's absolutely no point in fixing this weird warning. Please drop this patch.

Reply:
Reworked the patch. Arguments now have more generic names.

Reply:
This particular SVACE warning makes no sense (as many others, actually). It just checks that function arguments aren't swapped by mistake by looking at the argument names. Say, we have function `foo(int x, int y)`, then if you call it as `foo(y, x)` somewhere in the code, it'll trigger this warning while `foo(x, y)` will pass the check as well as `foo(a, b)`.

It's pointless to suppress this warning by swapping or renaming arguments, because there's nothing that prevents us from writing the code like that in future: tests will pass either way; `checkpatch` won't complain; a reviewer usually won't insist on a particular argument order. We have marked literally all `WRONG_ARGUMENTS_ORDER` errors as FP (and we have more than 30 of them). There's absolutely no point in going back there and fixing them. It'd be a waste of time.

Reply:
I dropped the patch as both reviewers think it's unnecessary.